src/pages/resource/ReportResource.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, useState } from 'react';
+import ReportProblemIcon from '@material-ui/icons/ReportProblem';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { DialogActions, DialogContent, TextField, useMediaQuery } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import ErrorBanner from '../../components/ErrorBanner';
+import { useTranslation } from 'react-i18next';
+import { DeviceWidths } from '../../themes/mainTheme';
+import { reportResource } from '../../api/workListApi';
+import styled from 'styled-components';
+import { Resource } from '../../types/resource.types';
+
+const StyledButton = styled(Button)`
+  min-width: 10rem;
+`;
+
+interface ReportResourceProps {
+  resource: Resource;
+  setRequestSentSuccess: (success: boolean) => void;
+}
+
+const ReportResource: FC<ReportResourceProps> = ({ resource, setRequestSentSuccess }) => {
+  const { t } = useTranslation();
+  const [showReportDialog, setShowReportDialog] = useState(false);
+  const [reportText, setReportText] = useState('');
+  const [sendReportError, setSendReportError] = useState<Error>();
+  const useFullScreen = useMediaQuery(`(max-width:${DeviceWidths.sm}px)`);
+
+  const handleReport = async () => {
+    setSendReportError(undefined);
+    setRequestSentSuccess(false);
+    try {
+      await reportResource(resource.identifier, reportText);
+      setShowReportDialog(false);
+      setRequestSentSuccess(true);
+    } catch (error) {
+      setSendReportError(error);
+    }
+  };
+
+  const cancelReport = () => {
+    setShowReportDialog(false);
+    setReportText('');
+  };
+  return (
+    <>
+      <StyledButton
+        data-testid={`report-resource-button`}
```

Comment:
pirk: trenger ikke `{ }` når det bare en en streng, ser det gjelder flere props nedover her også
```suggestion
        data-testid="report-resource-button"
```